pkg/provisioner/ironic/validatemanagementaccess_test.go

```diff
@@ -836,7 +836,7 @@ func TestPreprovisioningImageFormats(t *testing.T) {
 			Name:              "enabled virtualmedia",
 			Address:           "redfish-virtualmedia://example.test",
 			PreprovImgEnabled: true,
-			Expected:          []metal3v1alpha1.ImageFormat{"iso", "initrd"},
+			Expected:          []metal3v1alpha1.ImageFormat{"initrd", "iso"},
```

Comment:
This reordering means that initrd will have a priority over iso, which I think is not intended for virtual media.

Reply:
Ah you're right, I haven't caught that one because the coupling is really loose between the controllers.